### ISSUE TITLE
Saving a new Sass file defaults to ".haml" as the extension

### ISSUE THREAD

**Issue by svivian** [nathos/sass-textmate-bundle#99]
To reproduce:

1. Start a new file, write some Sass/CSS.
2. Select Sass as the current syntax (or alternatively, copy-paste some Sass from another file, it will set that automatically).
3. Save the file.

When the save dialog comes up, the suggested filename defaults to `untitled.haml`. It should be `untitled.scss`.

This is on a MacBook Pro, OSX 10.10.5. Sublime Text 3, Build 3126 (stable channel).

I've followed the README but I think this is a slightly different issue. ST interprets the files as Sass just fine and colours them so, it's just when saving a new file it defaults to a `.haml` extension. As noted in #41 in ST3 there are no separate packages available, they're built into ST. The only `tmLanguage` files on my entire system are the binary files in the Cache folder like `~/Library/Application Support/Sublime Text 3/Cache/Rails/Ruby Haml.tmLanguage.cache`

**Comment by hey-stefan** [comment]
Hi @svivian – have you been able to solve this? Tried everything, still can't in 2017. :)

**Comment by svivian** [comment]
@stefan-t nope, still have the same problem to this day. I also have a similar issue with Laravel Blade templates, it gives `.blade` as the extension when it should be `.blade.php`

Not sure if it's the plugin itself or if there is a deeper issue with Sublime.

**Comment by hey-stefan** [comment]
@svivian what's worse is, I must rename the file after it's saved, because if i remove ".haml" from the dialog box, it reappears in the filename itself... So frustrating.

**Comment by svivian** [comment]
What platform are you on? On Mac I get a popup asking if I'm sure I want to save the file with that extension, but it does save it with the correct ext. From recollection it worked fine on Linux last time I used it.

**Comment by dsasko** [comment]
Try installing the SCSS package.
Before saving the new file make sure the syntax in sublime is set to SCSS.

This solved the issue for me.

OSX - 10.11.6
ST3 - build 3126

**Comment by svivian** [comment]
@dsasko OK, but that's a different package. It doesn't solve the issues with this one.
